src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/__tests__/ExposureSubmissionCoordinatorTests.swift

```diff
@@ -32,7 +32,10 @@ class ExposureSubmissionCoordinatorTests: XCTestCase {
 
 	// MARK: - Setup and teardown methods.
 
-	override func setUp() {
+	private var store: SecureStore!
+	
+	override func setUpWithError() throws {
+		store = try SecureStore(at: URL(staticString: ":memory:"), key: "123456", serverEnvironment: ServerEnvironment())
```

Comment:
We have a `MockTestStore()` for unit tests.

Reply:
Also valid for other adjusted tests.

Reply:
Fixed